### ISSUE TITLE
Let me enter error description before/while starting error data collection

### ISSUE THREAD

**Issue by rugk** [abrt/libreport#555]
I have a small UX problem:
* When I see a crash catched by abrt, I usually click on "Report". I often know, what I've just done that caused this. 
* Then abrt is collecting some data or so…
* Still collection…
   I continue doing something else.
* It seems to ask me the usual question whether to upload data.
* Abrt works…
* …
* At the end, it finally shows me a text field, where I can input the error description/should describe what I have just done.

The problem now is: I have already forgotten that…

So **User Story:** I, as a user of abrt, want to be able to describe the crash cause as fast as I can after it occurred, because I cannot remember this longer than 1 minute (:wink:).

## Solution

Let me enter the error description _before_ doing these long operations on collecting data.

Or, alternatively, (as AFAIK the data collecting can also result in a crash being non-reportable) do it at the same time. Either split the window or just do data collection in the background.
This way, I am not losing time describing a issue I possibly cannot report anyway.

**Comment by jfilak** [comment]
Good idea, but I am not sure if you wouldn't be disappointed by the fact that ABRT often does not let you report the detected problem because of low quality of collected data (mainly backtrace) or it might ignore your comments when a duplicate is found - and unfortunately, at the moment (if I remember it correclyt) you must ask ABRT to start collecting data because it might consume lot of computer resources - downloading all debuginfos and so on. Perhaps, a warning saying "please, be warned that ABRT might not let you send your comments", would help.

**Comment by rugk** [comment]
> when a duplicate is found

AFAIK it still adds the text as a comment then, anyway.
